src/platform_impl/macos/util/mod.rs

```diff
@@ -113,31 +104,6 @@ pub unsafe fn ns_string_id_ref(s: &str) -> IdRef {
     IdRef::new(NSString::alloc(nil).init_str(s))
 }
 
-/// Returns the number of characters in a string.
-/// (A single character may consist of multiple UTF-32 code units.
-/// This is possible when long sequences of composing characters are present)
-///
-/// Unsafe because assumes that the `string` is an `NSString` object
-pub unsafe fn ns_string_char_count(string: id) -> usize {
-    let length: NSUInteger = msg_send![string, length];
-    let range = NSRange {
-        location: 0,
-        length,
-    };
-    let char_count = Rc::new(Cell::new(0));
-    let block = {
-        let char_count = char_count.clone();
-        ConcreteBlock::new(move || char_count.set(char_count.get() + 1)).copy()
```

Comment:
This was also UB btw, the block must take the four arguments as described in the [docs](https://developer.apple.com/documentation/foundation/nsstring/1416774-enumeratesubstringsinrange?language=objc) (even if unused)

Reply:
Thanks for pointing this out, I'll try to remember this for the future.